next.config.js

```diff
@@ -16,6 +16,6 @@ module.exports = {
     return config;
   },
   env: {
-    GA_TRACKING_ID: process.env.GA_TRACKING_ID,
+    NEXT_PUBLIC_GA_TRACKING_ID: process.env.NEXT_PUBLIC_GA_TRACKING_ID,
```

Comment:
I think you can avoid this `env` config altogether now assuming this is added to `NEXT_PUBLIC_GA_TRACKING_ID` in your Vercel project.